ompi/mca/coll/xhc/README.md

```diff
@@ -202,17 +202,17 @@ Apart from instructing Open MPI to include XPMEM support, the rest of the build
 process is standard. General information on building Open MPI can be found in
 its documentation.
 
-<https://www.open-mpi.org/doc/>  
-<https://www.open-mpi.org/faq/?category=building>  
+<https://www.open-mpi.org/doc/>
+<https://www.open-mpi.org/faq/?category=building>
```

Comment:
Hey, here's a comment: these double trailing spaces (here and in other spots in the same file) act as line breaks. Perhaps it would be better to replace them with `\`, to avoid similar future issues with automatic trimmers.

Reply:
Oh wow, I had no idea that 2 trailing spaces in a markdown file was supposed to be a line break.  I'll fix.